configure.ac

```diff
@@ -278,5 +278,7 @@ AC_OUTPUT
 echo "*"
 echo "* AWS OFI NCCL plugin has been configured."
 echo "*"
-echo "* Platform-specific optimizations: ${NCCL_OFI_PLATFORM}"
+AS_IF([test "${NCCL_OFI_PLATFORM}" = "none"],
+      [echo "* Platform Optimizations: DISABLED"],
+      [echo "* Platform Optimizations: ${NCCL_OFI_PLATFORM}"])
```

Comment:
This isn't quite what I asked for.  "DISABLED" is really wrong, since default will still be built.  Just build up a list of the enabled platforms (including the default plugin, of course).